src/predef.js

```diff
@@ -103,6 +97,9 @@ module.exports = {
         dependencies: [],
         code: [
             'function $has(obj, key) {',
+            '    if (obj === null || obj === undefined) {',
+            '        return false;',
```

Comment:
I think this should throw an error about trying to check keys of null or undefined.